lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
+      'sessionTokenVerified',
```

Comment:
My understanding is that adding new auth strategies will require updates to clients to create Hawk credentials using `sessionTokenVerified` instead of `sessionToken`. One of the goals was to create backwards-compatible client changes, updating `/recovery_email/status` to use the  [sessionTokenVerified strategy as rfk mentions](https://github.com/mozilla/fxa-auth-server/pull/1232/files/fc6b6c9a2789a687d00d7396d2cd544317bd1188#r60522700) will break existing clients. Same with `recovery_email/resend_code` if [it's updated to sessionTokenVerified as I mention in the db PR](https://github.com/mozilla/fxa-auth-db-mysql/pull/132/files#r60721355).